scripts/upgrade_schema/update_schema_utility

```diff
@@ -0,0 +1,86 @@
+--
+-- Tables structures required by utility feature (uqc)
+--
+DROP TABLE IF EXISTS `uqc_outcome_hist`;
+DROP TABLE IF EXISTS `uqc_outcome_ent`;
+DROP TABLE IF EXISTS `uqc_outcome_dict`;
+--
```

Comment:
This is a script for changing the production database. If the tables already exist they cannot be dropped  because they can contain data.